src/DebugBar/Resources/debugbar.js

```diff
@@ -1293,4 +1293,88 @@ if (typeof(PhpDebugBar) == 'undefined') {
 
     });
 
+    // ============= Handling Dragging Behavior ============
+    var storedPosX = localStorage.getItem('phpdebugbarPositionX') || 0;
+    var screenWidth = $(window).width();
+
+    $(document).on('click', '.phpdebugbar', function () {
+        let isClosed = $('.phpdebugbar').hasClass('phpdebugbar-closed');
+        if (!isClosed) {
+            // Save current position to localStorage when closing the debugbar
+            storedPosX = localStorage.getItem('phpdebugbarPositionX');
```

Comment:
maybe more consistent with the other storage keys
like `phpdebugbar-restore-position`
https://github.com/maximebf/php-debugbar/blob/c9cb28d5ee6f657fcc48440f38db8bbd140bef83/src/DebugBar/Resources/debugbar.js#L1151
https://github.com/maximebf/php-debugbar/blob/c9cb28d5ee6f657fcc48440f38db8bbd140bef83/src/DebugBar/Resources/debugbar.js#L790-L791